tests/feature_elimination/test_feature_elimination.py

```diff
@@ -10,7 +10,7 @@
 from sklearn.svm import SVC
 from xgboost import XGBClassifier, XGBRegressor
 
-from probatus.feature_elimination import EarlyStoppingShapRFECV, ShapRFECV
+from probatus.feature_elimination import ShapRFECV
```

Comment:
Shall we leave at least 1 test with the previous class, just to make sure it doesn't get broken before it's actually removed?